app/invocation/invocation_action_card.tsx

```diff
@@ -397,15 +401,21 @@ export default class InvocationActionCardComponent extends React.Component<Props
       .then((buffer) => {
         let dir = build.bazel.remote.execution.v2.Directory.decode(new Uint8Array(buffer));
         this.state.treeShaToExpanded.set(digestString, true);
-        let dirs: InputNode[] = dir.directories.map((child) => ({
+        let nodes: TreeNode[] = dir.directories.map((child) => ({
           obj: child,
           type: "dir",
         }));
-        let files: InputNode[] = dir.files.map((child) => ({
+        let files: TreeNode[] = dir.files.map((child) => ({
           obj: child,
           type: "file",
         }));
-        this.state.treeShaToChildrenMap.set(digestString, dirs.concat(files));
+        nodes.push(...files);
+        let symlinks: TreeNode[] = dir.symlinks.map((child) => ({
+          obj: child,
+          type: "symlink",
+        }));
+        nodes.push(...symlinks);
```

Comment:
same here - avoid using the spread operator in function calls

```suggestion
        const nodes = [...directories, ...files, ...symlinks];
```